docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Reactive_Streams_Operators_SmallRye.adoc

```diff
@@ -0,0 +1,53 @@
+[[MicroProfile_Reactive_Streams_Operators_SmallRye]]
+= MicroProfile Reactive Streams Operators Subsystem Configuration
+
+Support for https://microprofile.io/project/eclipse/microprofile-reactive-streams[Eclipse MicroProfile Reactive Streams Operators] is
+provided as a Tech Preview feature by the _microprofile-reactive-streams-operators-smallrye_ subsystem.
+
+[[required-extension-microprofile-reactive-streams-operators-smallrye]]
+== Required Extension
+
+This extension is *not* included in the standard configurations included in the WildFly distribution.
+
+You can add the extension to a configuration either by adding
+an `<extension module="org.wildfly.extension.microprofile.reactive-streams-operators-smallrye"/>`
+element to the xml or by using the following CLI operation:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /extension=org.wildfly.extension.microprofile.reactive-streams-operators-smallrye:add
+{"outcome" => "success"}
+
+[standalone@localhost:9990 /] /subsystem=microprofile-reactive-streams-operators-smallrye:add
+{
+    "outcome" => "success",
+    "response-headers" => {
+        "operation-requires-reload" => true,
+        "process-state" => "reload-required"
+    }
+}
+----
+
+If provisioning your own server and including the `microprofile-reactive-streams-operators` layer, you will get the required modules and the extension and subsystem will be added to your configuration.
```

Comment:
Note that all the new layers also need to be added to https://github.com/wildfly/wildfly/blob/master/docs/src/main/asciidoc/_galleon/Galleon_layers.adoc